types/index.test-d.ts

```diff
@@ -43,10 +43,6 @@ const OAuth2Options: FastifyOAuth2Options = {
     credentials: credentials,
     callbackUri: 'http://localhost/testOauth/callback',
     callbackUriParams: {},
-    generateStateFunction: () => {
```

Comment:
why this has been removed?

Reply:
it wasn't being used in any other place. Since it needs to receive a fastifyRequest, it essentially wasn't doing anything, but typescript was yelling, so I removed it!

Reply:
You still need the test for this two function.
Please update to the valid syntax.